static/src/javascripts/projects/common/modules/article/rich-links.js

```diff
@@ -19,6 +20,24 @@ const richLinkTag = ({ href }: { href: string }): string =>
         <p><a href="${href}">${href}</a></p>
     </aside>`;
 
+const hideIfPaidForAndAdFree = (el: Element): Promise<void> => {
+    if (!commercialFeatures.adFree) {
+        return Promise.resolve();
+    }
+    return fastdom.write(() => {
+        for (let child = 0; child < el.children.length; child += 1) {
+            if (
+                el.children[child].classList
+                    .toString()
+                    .includes('rich-link--paidfor')
+            ) {
+                el.classList.add('u-h');
+                break;
+            }
+        }
```

Comment:
You might be able to achieve something similar by spreading `el.children` into an Array and filtering:

```js
[...el.children]
  .filter(child => child.classList.toString().includes(...))
  .forEach(child => child.classList.add(...))
```

(untested code!)

Reply:
That looks much cleaner. I'll try it. Thanks!